src/images/svg/flexibility.svg

```diff
@@ -0,0 +1,85 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  xmlns:xlink="http://www.w3.org/1999/xlink"
+  width="500"
+  zoomAndPan="magnify"
+  viewBox="0 0 375 374.999991"
+  height="500"
+  preserveAspectRatio="xMidYMid meet"
+  version="1.0"
+>
+  <defs>
+    <clipPath id="198aceba12">
+      <path
+        d="M 29.035156 24.878906 L 345.835938 24.878906 L 345.835938 341.796875 L 29.035156 341.796875 Z M 29.035156 24.878906 "
+        clip-rule="nonzero"
+      />
+    </clipPath>
+    <clipPath id="6bea0ccb0b">
+      <path
+        d="M 29.035156 155 L 85 155 L 85 212 L 29.035156 212 Z M 29.035156 155 "
+        clip-rule="nonzero"
+      />
+    </clipPath>
+    <clipPath id="ca63622e55">
+      <path
+        d="M 140 24.878906 L 197 24.878906 L 197 81 L 140 81 Z M 140 24.878906 "
+        clip-rule="nonzero"
+      />
+    </clipPath>
+  </defs>
```

Comment:
this icon is broken (ss from PR), i think this part of svg does this
![image](https://github.com/vazco/forminer/assets/49162755/e1a81ab8-90b4-4188-a402-df863b388b8f)

Reply:
I need to obtain a new svg for this, the original one is just broken

Reply:
This might not be the svg's problem as some other svgs that I haven't touched become broken as well